styles/go.dev/catppuccin.user.css

```
@@ -88,7 +88,7 @@
     --color-background-card-footer: @crust;
     --color-background-code: @surface0; // Code Snippets
     --color-background-inverted: @crust;
-    --color-background-logo: @text;
+    --color-background-logo: @mantle;
```

should probably be surface0 since we are moving inwards towards the centre of the page, based on the style guidelines.
```suggestion
    --color-background-logo: @surface0;
```